src/components/Datepicker/Datepicker.js

```diff
@@ -0,0 +1,185 @@
+            onChange={this.viewYear}
+          >
+            {this.renderYearOptions()}
+          </select>
+        </div>
+      </div>
+    );
+  }
+
+  renderWeekHeader() {
+    return (
+      <thead>
+        <tr id="weekdays">
+          {moment.weekdaysShort().map(wd => <th key={wd} id={wd} tbody="col">
+            <abbr title={wd}>{wd}</abbr>
+          </th>
+          ) }
+        </tr>
+      </thead>
+    );
+  }
+
+  renderMonth() {
+    return (
+      <tbody>
+        {this.getMonthDays().map((week, weekIndex) => this.renderWeek(week, weekIndex))}
+      </tbody>
+    );
+  }
+
+  renderWeek(week, weekIndex) {
+    return (
+      <tr key={weekIndex}>
+        {week.map(([day, inRange], dayIndex) => this.renderDay(day, inRange, dayIndex))}
+      </tr>
+    );
+  }
+
+  renderDay(day, inRange, dayIndex) {
+    const classes = {
+      [this.prefix(['is-today'])]: day.isSame(moment(), 'day'),
+      [this.prefix(['disabled-text'])]: !inRange,
+      [this.prefix(['is-selected'])]: day.isSame(this.state.selectedDate, 'day'),
+    };
+    return (
+      <td key={dayIndex} className={classnames(classes)} headers={day.day()}>
+        <span className={this.prefix(['day'])} onClick={() => this.selectDay(day)}>
+          {day.date()}
+        </span>
+      </td>
+    );
+  }
+
+  render() {
+    return (
+      <div className={this.prefix(['datepicker', 'dropdown', 'dropdown--left'])}>
+        <div className={this.prefix(['datepicker__filter', 'grid'])}>
+          <div className={this.prefix(['datepicker__filter--month', 'grid', 'grid--align-spread', 'grow'])}>
+            <div className={this.prefix(['align-middle'])}>
+              <Button
+                icon
+                onClick={() => this.viewMonth(-1)}
+                className={this.prefix(['button', 'button--icon-container'])}
+              >
+                <ButtonIcon position="left" sprite="utility" icon="left" />
+              </Button>
+            </div>
+            <h2 id="month" className={this.prefix(['align-middle'])}>
+              {this.state.viewedDate.format('MMMM')}
+            </h2>
+            <div className={this.prefix(['align-middle'])}>
+              <Button
+                icon
+                onClick={() => this.viewMonth(1)}
+                className={this.prefix(['button', 'button--icon-container'])}
+              >
+                <ButtonIcon position="right" sprite="utility" icon="right" />
+              </Button>
+            </div>
+          </div>
+          {this.renderYearPicker()}
+        </div>
+        <table className={this.prefix(['datepicker__month'])} role="grid">
+          {this.renderWeekHeader()}
+          {this.renderMonth()}
+          <tr>
+            <td colSpan="7" role="gridcell">
+              <a
+                onClick={this.resetViewToToday}
+                className={this.prefix(['show--inline-block', 'p-bottom--x-small'])}
+              >
+               Today
```

Comment:
English string. We should also allow to inject the strings via prop and define the English strings as default prop